src/main/java/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public class BaseEntity {
```

Comment:
```suggestion
public abstract class BaseEntity {
```

추상 클래스라는 의도를 나타내보아요.